### ISSUE TITLE
[Bug]: Cannot search text in telescope with a fresh install

### ISSUE THREAD

**Issue by citypaul** [LunarVim/LunarVim#1326]
### Problem description

When I install LunarVim with a fresh install, I see an error and no search output when trying to use telescope to search text (via `<leader> s t`)

This was happening for me after I tweaked the config, so I decided to revert to a vanilla installation from the latest release and I am still seeing the issue even then, with no modifications on my end.

### LunarVim version

0.5.1

### Neovim version (>= 0.5)

0.5.0

### Relevant log output

```shell
## nvim -v


NVIM v0.5.0
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/config -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/src -I/usr/local/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I/usr/local/opt/gettext/include -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/src/nvim/auto -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/include
Compiled by brew@BigSur

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.5.0/share/nvim"

Run :checkhealth for more info


health#compe#check
========================================================================
## compe:snippet
  - OK: snippet engine detected.

## compe:mapping
  - OK: `compe#complete` is mapped: (`i  <C-Space>   * compe#complete()`)
  - INFO: `compe#confirm` is not mapped
  - OK: `compe#close` is mapped: (`i  <C-E>       * compe#close('<C-E>')`)
  - OK: `compe#scroll` is mapped: (`i  <C-D>       * compe#scroll({ 'delta': -4 })`)

health#which_key#check
========================================================================
## WhichKey: checking conflicting keymaps

health#nvim_treesitter#check
========================================================================
## Installation
  - OK: `tree-sitter` found  0.20.0 (parser generator, only needed for :TSInstallFromGrammar)
  - OK: `node` found v14.16.1 (only needed for :TSInstallFromGrammar)
  - OK: `git` executable found.
  - OK: `cc` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl" }
  - OK: Neovim was compiled with tree-sitter runtime ABI version 13 (required >=13). Parsers must be compatible with runtime ABI.

## Parser/Features H L F I J

 Legend: H[ighlight], L[ocals], F[olds], I[ndents], In[j]ections
         +) multiple parsers found, only one will be used
         x) errors found in the query, try to run :TSUpdate {lang}

health#lspconfig#check
========================================================================
## Checking language server protocol configuration
  - INFO: null-ls: configuration checked.

health#nvim#check
========================================================================
## Configuration
  - WARNING: Missing user config file: /Users/paul/.config/nvim/init.vim
    - ADVICE:
      - :help |init.vim|

## Performance
  - OK: Build type: Release

## Remote Plugins
  - OK: Up to date

## terminal
  - INFO: key_backspace (kbs) terminfo entry: key_backspace=^H
  - INFO: key_dc (kdch1) terminfo entry: key_dc=\E[3~
  - INFO: $TERM_PROGRAM='iTerm.app'
  - INFO: $COLORTERM='truecolor'

health#provider#check
========================================================================
## Clipboard (optional)
  - OK: Clipboard tool found: pbcopy

## Python 2 provider (optional)
  - WARNING: No Python executable found that can `import neovim`. Using the first available executable for diagnostics.
  - ERROR: Python provider error:
    - ADVICE:
      - provider/pythonx: Could not load Python 2:
          /usr/bin/python2 does not have the "neovim" module. :help |provider-python|
          /usr/bin/python2.7 does not have the "neovim" module. :help |provider-python|
          python2.6 not found in search path or not executable.
          /usr/bin/python does not have the "neovim" module. :help |provider-python|
  - INFO: Executable: Not found

## Python 3 provider (optional)
  - INFO: `g:python3_host_prog` is not set.  Searching for python3 in the environment.
  - INFO: Multiple python3 executables found.  Set `g:python3_host_prog` to avoid surprises.
  - INFO: Executable: /Library/Frameworks/Python.framework/Versions/3.8/bin/python3
  - INFO: Other python executable: /usr/local/bin/python3
  - INFO: Other python executable: /usr/bin/python3
  - INFO: Python version: 3.8.2
  - INFO: pynvim version: 0.4.1 (outdated; from /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/neovim)
  - WARNING: Latest pynvim is NOT installed: 0.4.3

## Python virtualenv
  - OK: no $VIRTUAL_ENV

## Ruby provider (optional)
  - INFO: Ruby: ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin20]
  - WARNING: `neovim-ruby-host` not found.
    - ADVICE:
      - Run `gem install neovim` to ensure the neovim RubyGem is installed.
      - Run `gem environment` to ensure the gem bin directory is in $PATH.
      - If you are using rvm/rbenv/chruby, try "rehashing".
      - See :help |g:ruby_host_prog| for non-standard gem installations.

## Node.js provider (optional)
  - INFO: Node.js: v14.16.1
  - WARNING: Missing "neovim" npm (or yarn) package.
    - ADVICE:
      - Run in shell: npm install -g neovim
      - Run in shell (if you use yarn): yarn global add neovim

## Perl provider (optional)
  - ERROR: perl provider error:
    - ADVICE:
      - "Neovim::Ext" cpan module is not installed

health#treesitter#check
========================================================================
## Checking treesitter configuration
  - INFO: Runtime ABI version : 13

## Messages output:

[telescope] [WARN  15:55:08] ...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:415: Failed with msg:  .../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105:
 rg: Executable not found
[telescope] stack traceback:
[telescope] ^I.../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105: in function 'new'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/finders.lua:94: in function 'finder'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:411: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410>
[telescope] ^I[C]: in function 'pcall'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:364>
[telescope] [WARN  15:55:08] ...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:415: Failed with msg:  .../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105:
 rg: Executable not found
[telescope] stack traceback:
[telescope] ^I.../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105: in function 'new'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/finders.lua:94: in function 'finder'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:411: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410>
[telescope] ^I[C]: in function 'pcall'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:364>
[telescope] [WARN  15:55:08] ...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:415: Failed with msg:  .../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105:
 rg: Executable not found
[telescope] stack traceback:
[telescope] ^I.../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105: in function 'new'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/finders.lua:94: in function 'finder'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:411: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410>
[telescope] ^I[C]: in function 'pcall'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:364>
[telescope] [WARN  15:55:08] ...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:415: Failed with msg:  .../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105:
 rg: Executable not found
[telescope] stack traceback:
[telescope] ^I.../site/pack/packer/start/plenary.nvim/lua/plenary/job.lua:105: in function 'new'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/finders.lua:94: in function 'finder'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:411: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410>
[telescope] ^I[C]: in function 'pcall'
[telescope] ^I...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:410: in function <...ck/packer/start/telescope.nvim/lua/telescope/pickers.lua:364>
```

### Screenshots

![image](https://user-images.githubusercontent.com/5850178/129482630-7d4348f2-0971-4f52-a0dd-8b3aff5dc7cc.png)

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by abzcoding** [comment]
can you install `ripgrep` and `fzf`  and test it again, it might be related to #1313

**Comment by citypaul** [comment]
Sure, I'll try that now, thanks

**Comment by citypaul** [comment]
Ok, so after I installed `ripgrep`, this worked for me.

I'm on MacOS 11.4 btw

**Comment by citypaul** [comment]
I just checked as well, and I already had `fzf` installed, just to let you know, as I'm not sure if I need both deps for this and whether there's an issue installing both with the install script or not.

**Comment by citypaul** [comment]
So just looking at the install script, and there is a function here that should install these packages:

https://github.com/LunarVim/LunarVim/blob/rolling/utils/installer/install.sh#L175-L178

This is called via the function `installExtraPackages`: https://github.com/LunarVim/LunarVim/blob/rolling/utils/installer/install.sh#L209-L218

But it looks like the `installExtraPackages` function itself isn't actually called in the script, so this would explain the issue I think?

**Comment by abzcoding** [comment]
yes we are aware of that, i'm closing this issue in favor of #1313 since they are the same, also this should be fixed via #1052

**Comment by citypaul** [comment]
Ah ok, thanks for your help!
